### PR TITLE
Add tsweeney's post.

### PR DESCRIPTION
**Pull request by sanjacloud**

### CODE REVIEW

File: source/blog/2018-01-16-buildah-blocks.html.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+I then modified the privileges on `runecho.sh` with:
+
+```
+# chmod +x runecho.sh
+```
+
+The [demo](https://asciinema.org/a/biE5aEvLRJK6uMBwd106eB7S1) shows the results of these steps.  I’ve included the commands used in the video at the bottom of this blog.  I will note that Red Hat is currently developing [Podman](https://github.com/projectatomic/libpod). Once that is fully developed, the `podman run` command will be the suggested way to run your container; the Buildah variant of that command will be positioned for quick testing purposes.
+
+I hope that you enjoyed the video and this article gives you a little more information on the Buildah project. Buildah is for simplicity - it is an Open Source project and contributors are always welcome. We’d love to see you there!
```
Comment (TomSweeneyRedHat) [inline source/blog/2018-01-16-buildah-blocks.html.md]: The "Buildah == Simplicity" is our tag line.  Can you change this second sentence to read  "Buildah is an Open Source..."  Then add "Buildah == Simplicity" on a new line after this paragraph?

File: source/blog/2018-01-16-buildah-blocks.html.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+done
+```
+
+I then modified the privileges on `runecho.sh` with:
+
+```
+# chmod +x runecho.sh
+```
+
+The [demo](https://asciinema.org/a/biE5aEvLRJK6uMBwd106eB7S1) shows the results of these steps.  I’ve included the commands used in the video at the bottom of this blog.  I will note that Red Hat is currently developing [Podman](https://github.com/projectatomic/libpod). Once that is fully developed, the `podman run` command will be the suggested way to run your container; the Buildah variant of that command will be positioned for quick testing purposes.
```
Comment (TomSweeneyRedHat) [inline source/blog/2018-01-16-buildah-blocks.html.md]: Change the first sentence to:
The [demo](https://asciinema.org/a/biE5aEvLRJK6uMBwd106eB7S1) shows an image built by Buildah being placed on Docker Hub and then run by both Docker and Buildah from there.

File: source/blog/2018-01-16-buildah-blocks.html.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+I hope that you enjoyed the video and this article gives you a little more information on the Buildah project. Buildah is for simplicity - it is an Open Source project and contributors are always welcome. We’d love to see you there!
+
+Commands used in the video for your perusal:
+
+```
+docker --version
+buildah --version
+docker login docker.io
+cat Dockerfile
```
Comment (TomSweeneyRedHat) [inline source/blog/2018-01-16-buildah-blocks.html.md]: I messed up, remove this line and the next two.  cat Dockerfile, ls -alF runecho.sh, cat runecho.sh.

File: source/blog/2018-01-16-buildah-blocks.html.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+ls -alF runecho.sh
+cat runecho.sh
+buildah bud -t echo .
+buildah push echo docker.io/tomsweeneyredhat/blog:echo
+buildah images
+docker images
+docker run docker.io/tomsweeneyredhat/blog:echo
```
Comment (TomSweeneyRedHat) [inline source/blog/2018-01-16-buildah-blocks.html.md]: Another mess up on my half.  Please move this docker run command up two lines above the "bulidah images" line.

**Comment by TomSweeneyRedHat** [comment]
@rhatdan FYI

**Comment by sanjacloud** [comment]
@TomSweeneyRedHat Please approve changes before I merge.

**Pull request merged**